worker/src/main/java/org/apache/celeborn/service/deploy/worker/congestcontrol/BufferStatusHub.java

```diff
@@ -71,4 +71,12 @@ public BufferStatusHub(int timeWindowsInSecs) {
   protected BufferStatusNode newEmptyNode() {
     return new BufferStatusNode();
   }
+
+  public long avgBytesPerSec() {
+    long currentNumBytes = sum().numBytes();
+    if (currentNumBytes > 0) {
+      return currentNumBytes * 1000 / (long) getCurrentTimeWindowsInMills();
+    }
+    return 0L;
```

Comment:
The if command seems not necessary?

Reply:
if `currentNumBytes` is 0, means there is no bytes being added, and `getCurrentTimeWindowsInMills` could be zero, causing the arithmetic exception

Reply:
> if `currentNumBytes` is 0, means there is no bytes being added, and `getCurrentTimeWindowsInMills` could be zero, causing the arithmetic exception

Got it, thanks